streamline_training.py

```diff
@@ -16,16 +16,68 @@
 import os
 import socket
 import subprocess
+import json
+
+from typing import Dict, Text
 
 """Train LaserTagger model and export to GCP bucket."""
 
 GCP_BUCKET = "gs://trained_models_yechen/"
 
 
+def write_lasertagger_config(output_dir: Text, bert_type: Text, t2t: bool, number_of_layer: int, hidden_size: int, attention_heads: int, filter_size: int, full_attention: bool):
```

Comment:
rename to export_lasertagger_config_to_json

Reply:
Done.